chromium_src/third_party/blink/public/mojom/permissions/permission.mojom

```diff
@@ -18,6 +18,7 @@ enum PermissionName {
   BRAVE_SPEEDREADER,
   BRAVE_ETHEREUM,
   BRAVE_SOLANA,
+  BRAVE_CARDANO,
```

Comment:
Is it ok to insert in the middle? Are underlying values of these enums persisted anywhere?